.github/workflows-source/ci-main.yml

```diff
@@ -276,6 +297,17 @@ jobs:
     if: github.repository == 'dfinity/ic'
     steps:
       - <<: *checkout
+      - <<: *generate-bazel-invocation-ids
+        with:
+          targets: |
+            [{
+              "name": "Test",
+              "output_id": "test_id",
+            }, {
+              "name": "Build",
+              "output_id": "build_id",
+            }]
```

Comment:
Would it be possible to only add this when `${{ needs.config.outputs.full_macos_build == 'true' }}`?